RELEASE.md

```diff
@@ -0,0 +1,38 @@
+# Release instructions
+
+The `pydata-sphinx-theme` follows semantic version numbers.
```

Comment:
I'm not sure I agree that we follow semantic versioning. If we think of `conf.py` / `html_theme_options` as our API, then we've made several breaking changes in my time working on the project, and yet we're still in version `0.x` (not `3.x` or `4.x` or whatever).  My feeling is that until we release a `1.0` we don't really want/need to commit to a versioning scheme, as there's generally an understanding that `0.x` software is rough around the edges.

```suggestion
```

That said: we should have a discussion about what it would take to get us to a point where we feel things are stable enough to bump to `1.0`. I'll open a separate issue about that.

Reply:
sure, I already open a milestone a long time ago and for me the most critical part is to stabilize the css prefixes 
https://github.com/pydata/pydata-sphinx-theme/milestone/11

Reply:
ok, great!  I'll start adding issues to that milestone.  But the point remains that I don't agree that we're following semantic versioning, so I think we should remove this line.